src/components/TransactionItem/index.tsx

```diff
@@ -75,6 +78,19 @@ const TransactionItem = ({
   const isXL = useIsExtraLarge()
   const { t } = useTranslation()
   const ref = useRef<HTMLDivElement>(null)
+  const txHash = transaction.transactionHash
+  // refactor to configurable
```

Comment:
```suggestion
  // TODO: refactor to configurable
```
Indexing will be enabled with conventional tag in some IDEs